applications/nrf5340_audio/doc/firmware_architecture.rst

```diff
@@ -141,13 +148,15 @@ Communication between modules is primarily done through Zephyr's :ref:`zephyr:zb
 
 The application uses the following buses:
 
-  * ``le_audio_chan`` - For handling LE Audio events from :file:`le_audio_cis_headset.c`, :file:`le_audio_bis_headset.c`, :file:`le_audio_cis_gateway.c`, or :file:`le_audio_bis_gateway.c`.
+  * ``le_audio_chan`` - For handling LE Audio events from the Bluetooth stream modules, specifically :file:`unicast_client.c`, :file:`unicast_server.c`, :file:`broadcast_source.c`, and :file:`broadcast_sink.c`.
```

Comment:
IMO, this is not something we should document:
- It is quite technical (i.e. for developers)
- It is quite well documented in the code
- There is overhead to maintain this doc.

Reply:
No action requested from me, just my two cents :)